bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/i18n/shelly_de.properties

```diff
@@ -300,6 +300,8 @@ channel-type.shelly.lastPower1.label = Schnitt letzte Minute
 channel-type.shelly.lastPower1.description = Stromverbrauch der letzten Minute, gerundet
 channel-type.shelly.meterTotal.label = Gesamtverbrauch
 channel-type.shelly.meterTotal.description = Gesamtverbrauch in kWh seit das Gerät eingeschaltet wurde (wird beim Neustart zurückgesetzt)
+channel-type.shelly.totalKWH.label = GesamtkWh (wird beim Neustart zurückgesetzt)
+channel-type.shelly.totalKWH.description = Gesamtverbrauchte Energie in kWh (wird beim Neustart zurückgesetzt)
```

Comment:
Please revert changes to this file. Translations are created in crowdin. 
For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations